src/clj/ataru/virkailija/virkailija_routes.clj

```diff
@@ -392,6 +392,15 @@
                                        (let [koodi-options (koodisto/get-koodisto-options koodisto-uri version)]
                                          (ok koodi-options))))
 
+                 (api/context "/organization" []
+                   :tags ["organization-api"]
+                   (api/GET "/hakukohderyhmat" []
+                     :return [ataru-schema/Hakukohderyhma]
+                     (->
+                       (.get-hakukohde-groups organization-service)
+                       ok
+                       (header "Cache-Control" "public, max-age=300"))))
```

Comment:
Onkohan meillä ollut tapana asettaa näitä cache-controleja eksplisiittisesti yksittäisille endpointeille?

Reply:
Mä vähän niinku alotin sen, kun halusin tuolla `/tarjonta` kontekstissa olevien endpointtien tuloksia selainkakkuun. Muutenhan meillä on middleware joka asettaa aina no-cache otsakkeen kaikkiin API endpointteihin.